code/unclassified/lapindrom_checker/lapindrome_checker.cpp

```diff
@@ -0,0 +1,35 @@
+// part of cosmos by opengenus foundation
+
+#include <iostream>
+#include <algorithm>   // used for sort()
+using namespace std;
+
+bool is_lapindrome(string str)
```

Comment:
If you want, you could replace `string` with a template, and a comparison template parameter,
this utility will be more powerful and useful!

Example is more power than words, e.g.,
```
template<typename _Container = std::string,
         typename _Comparison = std::less<typename std::iterator_traits<_Container>::value_type>>
bool isLapindrome(_Container serial)...
```

Reply:
@iattempt lapindrome is checked on strings only but not yet included in string algorithms so i have added here with `string` type